server/admin_management/api/app/main.py

```diff
@@ -67,9 +66,14 @@ def custom_generate_unique_id(route: APIRouter):
 )
 app.include_router(
     router_application_admin.router,
-    prefix=apiPrefix + "/application_admin",
+    prefix=apiPrefix + "/application_admins",
```

Comment:
I slightly adjust this to follow the REST path convention.

Reply:
okay, no problem